### ISSUE TITLE
allow multiple simultaneous collaborators on a subtitle / translate task

### ISSUE THREAD

**Issue by jdragojevic** [pculture/unisubs#902]
This is a longer-term feature request for ondemand teams.

When we have a translation job for longer videos, we need a way to allow more than 1 user to work on it at 1 time.

Ideally - we'd have a way to allow another team member to request to join in on the language, and have a way for two people to simultaneously edit the subs, similar to the way google doc editing or html pad editing works.  

Possibly would have to disable syncing to avoid conflict, or perhaps we can add a way in the ui to only sync a particular block of a video

Double-extra better would be to allow communication as they work via webrtc.

A secondary approach would be to break the job into chunks and the reconstruct it for the review phase.

**Comment by bendk** [comment]
This is implemented.

**Comment by PCF-Testing** [comment]
Not exactly. We have the option in collab team UI to request multiple endorsements from reviewers and approvers (although there is nether a proper spec nor good implementation for how the workflow should function in multiple-endorsements case), but we do not allow multiple subtitlers to work on the same collab concurrently.
